src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -10,65 +10,80 @@
 import java.sql.DriverManager;
 import java.sql.SQLException;
 import java.sql.Statement;
-import java.util.Date;
 
 import static junit.framework.TestCase.assertFalse;
```

Comment:
Why you're using this package for assertFalse? It should be org.junit.Assert.assertFalse

Reply:
Good remark, I'll fix it.